### DISCUSSION TITLE
How to use an alias of describe?

### DISCUSSION THREAD

**Discussion by dancork** [vitest-dev/vscode#327] [Q&A] [answered]
We are using context as a global alias of describe. The test explorer ui is not parsing this as a node in the testing tree though, has anyone else experienced this and have a solution for it?

**Comment by sheremet-va** [comment, marked as answer]
Do you use the latest pre-release version?

**Reply by dancork** [reply]
I don't but upgrading to pre-release has fixed it, thank :)
